source/movie.rb

```diff
@@ -0,0 +1,13 @@
+require './item'
+class Movie < Item
+  def initialize(*all, _silent)
+    super(*all)
```

Comment:
it would be nice if you could explicitly pass the date as an argument in the constructor instead of '*all'  since we already know the exact argument to pass when initialising the object